pkgs/by-name/ov/overskride/package.nix

```
@@ -0,0 +1,57 @@
+{ lib, fetchFromGitHub, stdenv, rustPlatform, cargo, rustc, meson, ninja
+, pkg-config, wrapGAppsHook4, desktop-file-utils, appstream-glib
+, blueprint-compiler, dbus, gtk4, libadwaita, bluez, libpulseaudio }:
+let
+
+  pname = "overskride";
```

Or if you do prefer this pattern of using string interpolation, do this

```suggestion
  pname = "overskride";
  repo = "overskride";
```